extensions/palette-colors/src/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from "react";
+import { ActionPanel, Action, Grid } from "@raycast/api";
+import { Theme, Color, BackgroundColor } from "@adobe/leonardo-contrast-colors";
+import ColorScheme from "color-scheme";
+
+const random = new BackgroundColor({
+  name: "random",
+  colorKeys: ["#cadada"],
+});
+
+const generateColors = (schemeType = "mono"): string[] => {
+  const scheme = new ColorScheme();
+  scheme
+    .from_hue(Math.floor(Math.random() * 360))
+    .scheme(schemeType)
+    .variation("soft");
+
+  const colors = scheme.colors().map((color: string) => {
+    return new Color({
+      name: "color",
+      colorKeys: [color],
+      ratios: [3, 4.5],
+    });
+  });
+
+  const theme = new Theme({ colors, backgroundColor: random, lightness: 7 });
+
+  return Array.from(new Set<string>(theme.contrastColorValues)).sort();
+};
+
+export default function Command() {
+  const [type, setType] = useState("mono");
+  const [colors, setColors] = useState(generateColors(type));
+
+  useEffect(() => {
+    setColors(generateColors(type));
+  }, [type]);
+
+  return (
+    <Grid
+      inset={Grid.Inset.Large}
```

Comment:
```suggestion
```
I would remove this to make the colors fill the grid 😁

Reply:
Good input, thank you very much! 👍🏻